debian/changelog

```diff
@@ -0,0 +1,12 @@
+eos-data-distribution (0.1.0) UNRELEASED; urgency=medium
```

Comment:
Please put debian-specific stuff onto the `debian-master` branch. Our CI system knows how to merge `debian-master` and `master` together when building.